docs/docs/user/hermes-mock.md

```diff
@@ -0,0 +1,105 @@
+# Hermes Mock
+
+A thin library designed to mocking Hermes in tests.
```

Comment:
I would clarify that we allow to mock publishing on Hermes frontend (not management operations, consumers etc).